src/cadenza.js

```diff
@@ -204,6 +204,8 @@ export class CadenzaClient {
    *   view, each row includes the geometry of the selected object as the last value.
    *   <p>
    *   See also: [JSON Representation of Cadenza Object Data](../index.html#md:json-representation-of-cadenza-object-data)
+   * @fires `change:extent` - When the map extent is changed a POST message 'change:extent' is fired.
+   *   The event details contain the new {@link Extent} of the map (transformed to EPSG:4326).
```

Comment:
On Friday, I improved the way how events are documented: https://github.com/DisyInformationssysteme/cadenza.js/commit/6020ae81bb42e257be3a87361f31d2708ef20b38

Goals:
- Remove the duplication of the documentation if the event is fired from multiple methods (like in your case).
- Document the type of the event detail.

Proposal for your event:
```javascript
/** @typedef {CadenzaEvent<'change:extent', {extent: Extent}>} CadenzaChangeExtentEvent - When the user moved the map. The extent is transformed according to the `useMapSrs` option. */
```

(Sry, I should've informed you about the change. I hope I didn't cause too much trouble.)